lib/rubocop/cop/rails/bulk_change_table.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'yaml'
```

Comment:
Can you remove it?

Reply:
We need it for `YAML.safe_load`? Is it required somewhere else?